src/main/java/seedu/address/logic/parser/patient/DeletePatientCommandParser.java

```diff
@@ -13,18 +16,41 @@
  */
 public class DeletePatientCommandParser implements Parser<DeletePatientCommand> {
 
+    /**
+     * Used for the separation of force delete format and patient index.
+     */
+    private static final Pattern FORCE_DELETE_FORMAT = Pattern.compile("(?<forceDelete>\\D+)(?<patientIndex>.*)");
```

Comment:
does this imply that force delete must be before patient index? ie. command must be given as `delete-patient --force 1`

if it does, is it intended?
if yes, then i may be better to specify maybe in the force delete message

Reply:
Yes, it is intended to be in the format of `delete-patient --force INDEX`. Will add in the javadocs.

Reply:
Yes, it is intended to be in the format of `delete-patient --force INDEX`, which is indicated in `FORCE_DELETE_MESSAGE_USAGE`. Will edit the javadocs so that it's clearer.